.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,7 +6,17 @@ about: Create a report to help reproduce and fix the issue
 
 ## Issue Summary
 
-<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. -->
+<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. Screenshots can be especially helpful. -->
+
+<!-- Please help cull duplicate and confusing issues. Your answers here are not required unless specifically marked as such. -->
+
+## Checklist
+
+[] Yes, I have searched for this issue. (REQUIRED)
+[] This occurs every time.
+[] This occurred once.
+[] The issue affects something in BYOND.
```

Comment:
This is meaningless to someone reporting bugs. What is "something in BYOND"? If something like the chat window is broken, they're probably going to click this when it's unlikely to be a BYOND issue. I think we shouldn't have this at all.